src/scales/scale.logarithmic.js

```diff
@@ -118,13 +118,13 @@ module.exports = Scale.extend({
 
 					helpers.each(dataset.data, function(rawValue, index) {
 						var values = valuesPerStack[key];
-						var value = +me.getRightValue(rawValue);
+						var value = me._parseValue(rawValue);
 						// invalid, hidden and negative values are ignored
-						if (isNaN(value) || meta.data[index].hidden || value < 0) {
+						if (isNaN(value.min) || isNaN(value.max) || meta.data[index].hidden || value.max < 0 || value.max < 0) {
```

Comment:
There's a double check here for `value.max < 0` which I think was meant to be `value.min < 0 || value.max < 0`

Reply:
done, nice catch! Thx!